### PR TITLE
Make SSEMessageFormatter internal and use new InternalsVisibleTo pattern

### PR DESCRIPTION
**Pull request by mikaelm12**
Updating to use the new `InternalsVisibleTo` tag in  csprojs rather than doing it via assembly info.

Also updated the `ServerSentEventsMessageFormatter` to be internal

### CODE REVIEW

**Review by davidfowl — APPROVED** [review]
Very nice

File: src/SignalR/clients/csharp/Http.Connections.Client/src/Microsoft.AspNetCore.Http.Connections.Client.csproj [hunk 1]
```diff
@@ -19,4 +19,15 @@
...
 
+  <ItemGroup>
+    <InternalsVisibleTo Include="Microsoft.AspNetCore.SignalR.Microbenchmarks" />
+    <InternalsVisibleTo Include="Microsoft.AspNetCore.SignalR.Tests" />
+    <InternalsVisibleTo Include="Microsoft.AspNetCore.SignalR.Client.Tests" />
+    <InternalsVisibleTo Include="DynamicProxyGenAssembly2" Key="$(MoqPublicKey)" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <Folder Include="Properties\" />
```
Comment (BrennanConroy) [inline src/SignalR/clients/csharp/Http.Connections.Client/src/Microsoft.AspNetCore.Http.Connections.Client.csproj]: Remove?

File: src/SignalR/Directory.Build.props [hunk 2]
```diff
@@ -6,12 +6,17 @@
...
+    <MoqPublicKey>0024000004800000940000000602000000240000525341310004000001000100c547cac37abd99c8db225ef2f6c8a3602f3b3606cc9891605d02baa56104f4cfc0734aa39b93bf7852f7d9266654753cc297e7d2edfe0bac1cdcf9f717241550e0a7b191195b7667bb4f64bcb8e2121380fd1d9d46ad2d92d2d15605093924cceaf74c4861eff62abf69b9291ed0a340e113be11e6a7d3113e92484cf7045cc7</MoqPublicKey>
+  </PropertyGroup>
+
   <PropertyGroup Condition="'$(IsTestProject)' != 'true'">
     <NoWarn>$(NoWarn);CS1591</NoWarn>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
     <PackageTags>aspnetcore;signalr</PackageTags>
   </PropertyGroup>
-
+
```
Comment (BrennanConroy) [inline src/SignalR/Directory.Build.props]:
```suggestion

```

**Review by BrennanConroy — APPROVED** [review]

File: src/SignalR/clients/csharp/Http.Connections.Client/src/Microsoft.AspNetCore.Http.Connections.Client.csproj (hunk 1 above)
Comment (mikaelm12) [reply src/SignalR/clients/csharp/Http.Connections.Client/src/Microsoft.AspNetCore.Http.Connections.Client.csproj]: Yeah, these kept getting added automatically.  I guess I missed one.

**Comment by mikaelm12** [comment]
@aspnet-hello triage https://dev.azure.com/dnceng/public/_build/results?buildId=231092&view=logs

**Comment by aspnet-hello** [comment]
This comment was made automatically. If there is a problem contact aspnetcore-build@microsoft.com.

I've triaged the above build. I've created/commented on the following issue(s)
https://github.com/aspnet/AspNetCore-Internal/issues/2667

**Pull request merged**